node/runtime/pangoro/src/pallets/evm.rs

```diff
@@ -88,6 +92,9 @@ where
 			a if a == addr(27) => Some(<StateStorage<R, StorageFilter>>::execute(
 				input, target_gas, context, is_static,
 			)),
+			a if a == addr(2048) =>
```

Comment:
Are these two precompiles only needed in the pangoro network?

Reply:
@hujw77 Do you need them in the Pangolin network?

Reply:
I think there is no need, the two percompiles is used for darwinia <==> eth2.0 and darwinia <==> bsc.